demisto_sdk/commands/validate/initializer.py

```diff
@@ -482,9 +482,15 @@ def git_paths_to_basecontent_set(
                     obj.git_status = git_status
                     # Check if the file exists
                     if git_status in (GitStatuses.MODIFIED, GitStatuses.RENAMED):
-                        obj.old_base_content_object = BaseContent.from_path(
-                            old_path, git_sha=git_sha, raise_on_exception=True
-                        )
+                        try:
+                            obj.old_base_content_object = BaseContent.from_path(
+                                old_path, git_sha=git_sha, raise_on_exception=True
+                            )
+                        except (NotAContentItemException, InvalidContentItemException):
+                            logger.debug(
+                                f"Could not parse the old_base_content_object for {str(obj.path)}, setting a copy of the object as the old_base_content_object."
```

Comment:
```suggestion
                                f"Could not parse the old_base_content_object for {obj.path}, setting a copy of the object as the old_base_content_object."
```

when using f-string it automatically calls `__str__` of the object